build/lib/readme_check.sh

```diff
@@ -23,11 +23,17 @@ source "${SCRIPT_ROOT}/common.sh"
 
 RETURN=0
 SED=$(build::find::gnu_variant_on_mac sed)
+PROJECT=${1:-}
 
-for GIT_TAG_FILE in projects/*/*/GIT_TAG
+GIT_TAG_FILES="projects/*/*/GIT_TAG"
```

Comment:
does this work for projects whcih are released branched?

Reply:
I think even earlier, we weren't checking for release branched projects since for those, the README format is slightly different so for now, manual updates are required.

Reply:
Added support for release branched projects.